gotocompany/guardian/v1beta1/guardian.proto

```diff
@@ -1051,6 +1094,16 @@ message Approval {
   uint32 appeal_revision = 14;
 }
 
+message ApprovalSummary {
```

Comment:
can we just call this `summary`? looking to the content it's not so specific to approvals
```suggestion
message Summary {
```